File: WWW/DesktopModules/NVCMS.Events/Display/ViewPage/Main.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.IO;
using System.Text.RegularExpressions;
using DotNetNuke;
using DotNetNuke.Entities;
using DotNetNuke.Entities.Modules;
using DotNetNuke.Common;
using DotNetNuke.Entities.Portals;
using DotNetNuke.Common.Utilities;
using NVCMS.Modules.TinTuc;
namespace DesktopModules.TinTuc.ViewPage
{
    public partial class Main : PortalModuleBase
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            PortalModuleBase control = new PortalModuleBase();
            string fbclid = Request.QueryString["fbclid"];
            string sUrl = Request.RawUrl.Replace("?fbclid=" + fbclid, "");
            string sId = Ultis.GetRequestId2(sUrl);
            string srequest = "";
            if (sId !="")
            {
                srequest = Ultis.GetRequestName(sId);
                if (srequest !="")
                {
                    if (srequest == "d")
                    {
                        control = Page.LoadControl("/DesktopModules/NVCMS.Events/Display/ViewPage/Detail.ascx") as PortalModuleBase;
                    }
                    else
                    {
                        control = Page.LoadControl("/DesktopModules/NVCMS.Events/Display/ViewPage/Index.ascx") as PortalModuleBase;
                    }
                }
                else
                {
                    control = Page.LoadControl("/DesktopModules/NVCMS.Events/Display/ViewPage/Index.ascx") as PortalModuleBase;
                }
            }
            else
            {
                control = Page.LoadControl("/DesktopModules/NVCMS.Events/Display/ViewPage/Index.ascx") as PortalModuleBase;
                
            }
            control.ModuleConfiguration = ModuleConfiguration;
            plhNews.Controls.Add(control);
            
        }
    }
}

File: WWW/DesktopModules/NVCMS.Form/Display/Main.ascx.vb
Imports System
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DotNetNuke
Namespace NVCMS.Modules.Form
    Partial Class MainCustomeDisplay
        Inherits Entities.Modules.PortalModuleBase

        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Try
                'If Not IsPostBack Then
                Dim DynamicPage As String
                If Settings(BL.settingForm_MailStyle) <> "" Then
                    DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/style/" & Settings(BL.settingForm_MailStyle))
                Else
                    DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/style/Capstone.ascx")
                End If
                Dim objModule As Entities.Modules.PortalModuleBase = CType(Me.LoadControl(DynamicPage), DotNetNuke.Entities.Modules.PortalModuleBase)
                If Not objModule Is Nothing Then
                    objModule.ModuleConfiguration = Me.ModuleConfiguration
                    phDynamicPlaceHolder.Controls.Add(objModule)
                End If
                'End If
            Catch ex As Exception
                ProcessModuleLoadException(Me, ex)
            End Try
        End Sub

    End Class
End Namespace
